src/views/quiz/QuizCreation.vue

```diff
@@ -61,30 +66,32 @@ export default {
       quiz: {
         name: '',
         description: '',
-        questions: [],
-        image: {}
-      }
+        questions: []
+      },
+      image: {}
     }
   },
   methods: {
+    ...mapActions({
+      create: 'Quiz/create'
+    }),
     createQuiz () {
-      QuizService.create(this.quiz, this.image).then(() => {
+      this.create(this.quiz, this.image).then(() => {
         this.quiz = {
           name: '',
           description: '',
-          questions: ''
+          questions: []
         }
-        this.image = ''
-        this.shouldStartQuestionCreation = false
+        this.image = {}
       }).catch((error) => {
         alert(error)
```

Comment:
??? Where is error handling?

Reply:
As I said in previous PR's it will be implemented application wide, when i create api endpoint which will store logs. I don't have idea how i want to give info to user yet.

Reply:
So don't catch errors if you can't handle them. Instead place @todo comment.

Reply:
Fixed